src/include/daos_srv/rebuild.h

```diff
@@ -21,15 +21,15 @@
  * associated targets
  */
 typedef enum {
-	RB_OP_FAIL,
+	RB_OP_EXCLUDE,
```

Comment:
@jolivier23 not for this PR obviously but would this be better as an X macro in master?

Reply:
this is not RPC code, but used by pool and rebuild module to tell different rebuild options.